### PR TITLE
Add Clients section

### LINKED ISSUE #28: How about a section for alternative npm clients?
Like https://github.com/alexanderGugel/ied and https://github.com/rstacruz/pnpm

### PR DESCRIPTION
**Pull request by zckrs**
fix #28

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -104,6 +105,18 @@
 - [john](https://github.com/davej/john) - Make npm3's flat dependencies easier to find and sort.
 
 
+## Clients
+
+### Official
```
Comment (sindresorhus) [inline readme.md]: No need to have separate subsection. Just note the official one in its description.

Comment (zckrs) [inline readme.md]: Like this ?

---

## Table of Contents
- [Articles](#articles)
- [Tools](#tools)
- [Packages](#packages)
- [Alternatives](#alternatives)
- [Tips](#tips)
- [FAQ](#faq)
- [Community](#community)
- [Documentation](#documentation)
- [Support](#support)

## ...

## Alternatives
- [pnpm](https://github.com/rstacruz/pnpm) - Performant 'npm install'.
- [ied](https://github.com/alexanderGugel/ied) - Like npm, but faster - an alternative package manager for Node.

## ...

Comment (sindresorhus) [inline readme.md]: No, it should be `Clients`, just not subsections like `Official`.

Comment (zckrs) [inline readme.md]: Pushed.

Comment (zckrs) [inline readme.md]: Arf you already merge.

**Pull request closed without merging**